cmd/operator-sdk/cli/legacy.go

```diff
@@ -82,7 +82,7 @@ func GetCLIRoot() *cobra.Command {
 		new.NewCmd(),
 		olm.NewCmd(),
 		printdeps.NewCmd(),
-		run.NewCmd(),
+		run.NewCmdLegacy(),
```

Comment:
helm/Ansible still requiring run `operator-sdk run --local` with the new layout.